src/EmployeeManagement/EmployeeManagement.IntegrationTest/ControllerTests/EmployeeControllerTests.cs

```diff
@@ -23,16 +22,16 @@ public async Task ShouldGetEmployeeList_WhenCalled_GetAllEmployeesAsync()
         {
             //Arrange
             TestDBSampleData.InitializeDbForTests(_appFactory);
-            var employeeDto = _mapper.Map<List<EmployeesDTO>>(TestDBSampleData.FetchSeedingEmployees());
+            var employeeDto = _mapper.Map<List<EmployeeDTO>>(TestDBSampleData.FetchSeedingEmployees());
 
             //Act
             var actual = await _httpClient.GetAsync($"{ApiRoutes.BaseUrl}/{ApiRoutes.Employee}");
-            var result = await actual.Content.ReadFromJsonAsync<List<EmployeesDTO>>();
+            var result = await actual.Content.ReadFromJsonAsync<List<EmployeeDTO>>();
 
             //Assert
             actual.StatusCode.Should().Be(HttpStatusCode.OK);
             result.Should().NotBeNullOrEmpty();
-            result.Should().BeEquivalentTo(employeeDto, x => x.Excluding(y => y.Id));
+            result.Should().BeEquivalentTo(employeeDto, x => x.Excluding(x => x.Id).Excluding(x => x.Address));
```

Comment:
Why are you excluding the address here?

Reply:
Address field comes from a different table and is linked through Guid, which is database generated. So, not able to mock the address field with exact guid.

Reply:
Checking for address field now